lib/roo/base.rb

```diff
@@ -12,6 +12,7 @@ class Roo::Base
   TEMP_PREFIX = 'roo_'
   MAX_ROW_COL = 999_999.freeze
   MIN_ROW_COL = 0.freeze
+  INTEGER_CLASS = 0.class == Integer ? Integer : Fixnum
```

Comment:
Didn't we just switch to using Integer instead of branching on `0.class` since Fixnum always inherited from Integer? https://github.com/salsify/rails-multitenant/pull/15/files

Reply:
mostly because I didn't know that `Fixnum` was a subclass of `Integer` in all Rubies back when I did this. Change.